src/config/settings.py

```diff
@@ -200,7 +200,7 @@
 SOCIAL_AUTH_URL_NAMESPACE = 'social'
 
 # Make sure OAuth redirects use HTTPS, e.g. https://localhost/complete/twitter
-SOCIAL_AUTH_REDIRECT_IS_HTTPS = True
+SOCIAL_AUTH_REDIRECT_IS_HTTPS = False
```

Comment:
This should remain True. Maybe the console FT's can warn people that they need to make it false locally in order for the tests to pass?